cpp/example_code/transcribe/get_transcript.cpp

```diff
@@ -0,0 +1,144 @@
+		//TODO(User): Update to location of your .crt file.
+		config.caFile = "C:/curl/bin/curl-ca-bundle.crt";
+#endif
+		config.region = region;
+
+		TranscribeStreamingServiceClient client(config);
+		StartStreamTranscriptionHandler handler;
+		handler.SetOnErrorCallback([](const Aws::Client::AWSError<TranscribeStreamingServiceErrors>& error) {
+            std::cerr << "ERROR: " + error.GetMessage() << std::endl;
+			});
+		//SetTranscriptEventCallback called for every 'chunk' of file transcripted. Partial results are returned in real time.
+		handler.SetTranscriptEventCallback([&received](const TranscriptEvent& ev) {
+			for (auto&& r : ev.GetTranscript().GetResults()) {
+				if (r.GetIsPartial()) {
+                    std::cout << "[partial] ";
+				}
+				else {
+                    std::cout << "[Final] ";
+				}
+				for (auto&& alt : r.GetAlternatives()) {
+                    std::cout << alt.GetTranscript() << std::endl;
+				}
+			}
+			received.Release();
+			});
+
+		StartStreamTranscriptionRequest request;
+		request.SetMediaSampleRateHertz(8000);
+		request.SetLanguageCode(LanguageCode::en_US);
+		request.SetMediaEncoding(MediaEncoding::pcm); //wav and aiff files are PCM formats
+		request.SetEventStreamHandler(handler);
+
+		auto OnStreamReady = [](AudioStream& stream)
+		{
+			Aws::FStream file(FILE_NAME, std::ios_base::in | std::ios_base::binary);
+			if (!file.is_open()) {
+				std::cerr << "Failed to open " << FILE_NAME << '\n';
+			}
+			std::array<char,BUFFER_SIZE> buf;
+			int i = 0;
+			while (file)
+			{
+				file.read(&buf[0], buf.size());
+
+				if (!file)
+					std::cout << "File: only " << file.gcount() << " could be read" << std::endl;
+
+				Aws::Vector<unsigned char> bits{ buf.begin(), buf.end()};
+				AudioEvent event(std::move(bits));
+				if (!stream)
+				{
+                    std::cerr << "Failed to create a stream" << std::endl;
+					break;
+				}
+				//The std::basic_istream::gcount() is used to count the characters in the given string. It returns
+				//the number of characters extracted by the last read() operation.
+				if (file.gcount() > 0) {
+					if (!stream.WriteAudioEvent(event))
+					{
+                        std::cerr << "Failed to write an audio event" << std::endl;
+ 						break;
+					}
+				}
+				else {
+					break;
+				}
+				std::this_thread::sleep_for(std::chrono::milliseconds(25)); // Slow down since we are streaming from a file.
+			}
+			if (!stream.WriteAudioEvent(AudioEvent())) { // Per the spec, we have to send an empty event (i.e. without a payload) at the end.
+				std::cerr << "Failed to send an empty frame" << std::endl;
+			}
+			else {
+                std::cout << "Successfully sent the empty frame" << std::endl;
+			}
+			stream.flush();
+			std::this_thread::sleep_for(std::chrono::seconds (END_OF_STREAM_SLEEP_SECONDS)); // Workaround to prevent an error at the end of the stream for this contrived example.
```

Comment:
Long lines render badly in the snippeted output. I suggest a max of ~88 characters.

Reply:
I have change max from 120 to 88.